lib/api.py

```diff
@@ -2,66 +2,66 @@
     if app.openapi_schema:
         return app.openapi_schema
     openapi_schema = get_openapi(
         title="RocketPy Infinity-API",
-        version="1.0.0 ALPHA",
+        version="1.0.0 BETA",
         description=(
-        "<p style='font-size: 18px;'>RocketPy Infinity-API is a RESTful API for RocketPy, a rocket flight simulator.</p>"
-        "<br/>"
-        "<button style='background-color: #4CAF50; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer;'>"
-        "<a href='https://api.rocketpy.org/docs' style='color: white; text-decoration: none;'>Swagger UI</a>"
-        "</button>"
-        "<br/>"
-        "<button style='background-color: #008CBA; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer;'>"
-        "<a href='https://api.rocketpy.org/redoc' style='color: white; text-decoration: none;'>ReDoc</a>"
-        "</button>"
-        "<p>Create, manage, and simulate rocket flights, environments, rockets, and motors.</p>"
-        "<p>Currently, the API only supports SolidMotor (calisto as power_off/on_drag and Cesaroni as thrust_source) and TrapezoidalFins. We apologize for the limitation, but we are actively working to expand its capabilities soon.</p>"
-        "<p>Please report any bugs at <a href='https://github.com/RocketPy-Team/infinity-api/issues/new/choose' style='text-decoration: none; color: #008CBA;'>GitHub Issues</a></p>"
+            "<p style='font-size: 18px;'>RocketPy Infinity-API is a RESTful Open API for RocketPy, a rocket flight simulator.</p>"
+            "<br/>"
+            "<button style='background-color: #4CAF50; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer;'>"
+            "<a href='https://api.rocketpy.org/docs' style='color: white; text-decoration: none;'>Swagger UI</a>"
+            "</button>"
+            "<br/>"
+            "<button style='background-color: #008CBA; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer;'>"
+            "<a href='https://api.rocketpy.org/redoc' style='color: white; text-decoration: none;'>ReDoc</a>"
+            "</button>"
+            "<p>Create, manage, and simulate rocket flights, environments, rockets, and motors.</p>"
+            "<p>Currently, the API only supports TrapezoidalFins. We apologize for the limitation, but we are actively working to expand its capabilities soon.</p>"
+            "<p>Please report any bugs at <a href='https://github.com/RocketPy-Team/infinity-api/issues/new/choose' style='text-decoration: none; color: #008CBA;'>GitHub Issues</a></p>"
         ),
         routes=app.routes,
```

Comment:
Out of curiosity... How did you generate this description?

Reply:
Chat GPT, of course 😆 
This is a hack to enable a friendly switch between swagger and redoc.ly standards (I prefer redoc.ly to check API docs, but swagger to run a smoke test, for instance)